ifrs17/Import/Importers.ipynb

```diff
@@ -248,10 +248,10 @@
         {
             "cell_type": "code",
             "source": [
-                "async public Task CleanDatabaseAsync<T> (Expression<Func<T, bool>> filter = null) where T : class",
+                "public static async Task CleanIfrsTargetAsync<T> (this IDataSource dataSource, Expression<Func<T, bool>> filter = null) where T : class",
```

Comment:
Why is your default not "true" already? You can avoid this null check.

Reply:
because it is a Expression<Func it requires to be a const at run time and does not compile to have default = (Expression<Func<T, bool>>)(x => true)